var/sabath/assets/sabath/models/c/cosmoflow/requirements.txt

```diff
@@ -0,0 +1,7 @@
+pandas
+wandb
+horovod==0.19.*
```

Comment:
That's very specific version of Horovod. Why not just use any version?

Reply:
This is `requirements.txt` for Python's `pip3` utility for installing packages. I don't see why Conda or any other package manager should be involved.